boot/systems.go

```diff
@@ -28,9 +28,10 @@ import (
 	"github.com/snapcore/snapd/strutil"
 )
 
-// clearTryRecoverySystem removes a given candidate recovery system from the
-// modeenv state file, reseals and clears related bootloader variables.
-func clearTryRecoverySystem(dev Device, systemLabel string) error {
+// ClearTryRecoverySystem removes a given candidate recovery system from the
+// modeenv state file, reseals and clears related bootloader variables. An empty
+// system label can be passed when the boot variables state is inconsistent.
```

Comment:
is the case where systemLabel is empty tested?

Reply:
Added a test now